src/widgets/AppWindow.js

```diff
@@ -109,7 +109,7 @@ export default class AppWindow extends createjs.Container {
       parent: this,
       role: AccessibilityModule.ROLES.MAIN,
     });
-
+    this.accessible.addChild(this._contentArea);
```

Comment:
Why is this line needed?  I would have expected the object passed to `AccessibilityModule.register` including `parent: this,` would handle this.

Reply:
Yes. i forgot that.